src/app/connect-wallet-alchemy/connect-wallet-alchemy.component.html

```diff
@@ -0,0 +1,83 @@
+<div class="h-screen bg-gradient-to-b from-green-100 to-brand-turquoise">
+  <div class="p-4">
+    <div class="flex items-center justify-between gap-6">
+      <img src="../assets/venture-miner-logo.png" class="w-32 bg-wite" />
+      <div class="flex flex-row items-center gap-4 text-sm">
+        <span
+          class="p-4 font-bold text-white rounded bg-brand-turquoise"
```

Comment:
![image](https://github.com/Venture-Miner/w3fs/assets/111710515/75e914b5-8e23-40a5-8c15-912a86a8a7f1)

When I click down there it makes the button behave as if I was trying to type something in it. Very weird behaviour. Is this normal?